polyfills/Error/isError/detect.js

```diff
@@ -0,0 +1 @@
+"isError" in Error && "DOMException" in self && Error.isError(new DOMException('error'));
```

Comment:
```suggestion
"isError" in Error && "DOMException" in self && (function () {
	try {
		return Error.isError(new DOMException());
	} catch (_) {
		return "isError" in Error;
	}
}())
```

The `"isError" in Error` in the `catch` block is to ensure that the detect script can return `true` when the polyfill is applied in older browser versions.